PathPlanning/DynamicWindowApproach/dynamic_window_approach.py

```diff
@@ -232,29 +235,34 @@ def plot_robot(x, y, yaw, config):  # pragma: no cover
         plt.plot([x, out_x], [y, out_y], "-k")
 
 
-def main(gx=10.0, gy=10.0, robot_type=RobotType.circle):
+def main(gx=-5.0, gy=-7.0, robot_type=RobotType.circle):
```

Comment:
Could you please change your main function to main2 or something and keep previous main function in order to fix unittest.

Reply:
Hi Atsushi, I have tried to change the name of 'main' function into 'test_pr' and keep the original 'main' function unchanged, then the CI test indicate 'xxx was not covered by tests', which means the 'test_pr' function was not used, therefore i deleted the 'test_pr' function and only keep the original main function.

below is the data which cause robot stucked issue in original code (in case you want replace the data and test the code):
goal = [-5, -7]
ob =-1* np.array([[-1, -1],
               [0, 2],
               [2, 6],
               [2, 8],
               [3, 9.27],
               [3.79, 9.39],
               [7.25, 8.97],
               [7.0, 2.0],
               [3.0, 4.0],
               [6.0, 5.0],
               [3.5, 5.8],
               [6.0, 9.0],
               [8.8, 9.0],
               [5.0, 9.0],
               [7.5, 3.0],
               [9.0, 8.0],
               [5.8, 4.4],
               [12.0, 12.0],
               [3.0, 2.0],
               [13.0, 13.0]
               ])